stone-java-gradle-plugin/src/main/kotlin/com/dropbox/stone/java/StonePlugin.kt

```diff
@@ -37,50 +36,48 @@ public class StonePlugin : Plugin<Project> {
         } else {
             "generate${sourceSet.name.capitalize()}Stone"
         }
-        val stoneTaskProvider: TaskProvider<StoneTask> = project.tasks.register(taskName, StoneTask::class.java)
-        stoneTaskProvider.get().also { stoneTask ->
-            stoneTask.description = "Generate Stone Java source files for ${sourceSet.name}."
+
+        project.tasks.register(taskName, StoneTask::class.java){
+            description = "Generate Stone Java source files for ${sourceSet.name}."
 
             val routeWhitelistFilterPropName = "com.dropbox.api.${sourceSet.name}.routeWhitelistFilter"
             val routeWhitelistFilterValue: String? = project.properties[routeWhitelistFilterPropName] as String?
             routeWhitelistFilterValue?.let {
-                stoneTask.routeWhitelistFilter(it)
+                routeWhitelistFilter(it)
             }
 
             val specDirPropName = "com.dropbox.api.${sourceSet.name}.specDir"
             val specDirPropNameValue: String? = project.properties[specDirPropName] as String?
             val mySpecDir: String = specDirPropNameValue ?: "src/${sourceSet.name}/stone"
 
-            stoneTask.specDir(mySpecDir)
```

Comment:
now stoneTask is the receiver so we don't need to explicitly state it over and over